REQUIREMENTS.md

```diff
@@ -5,6 +5,27 @@ The igraph homepage is built using Jekyll. Install
 [Jekyll](https://jekyllrb.com) if you don't have it yet. Further requirements
 are listed below.
 
+You need so install the following RubyGems (e.g. using `gem install`):
```

Comment:
I wouldn't list all the dependencies; `gem install` takes care of installing all the required dependencies anyway, wouldn't it? I would simply say that the user needs to run `gem install` to install everything.

Reply:
Ah, good to know! I went through the whole list because `make` complained each time :)